node/rpc/src/lib.rs

```diff
@@ -32,21 +32,36 @@ pub trait CreditcoinApi<BlockHash> {
 
 	#[pubsub(subscription = "events", unsubscribe, name = "creditcoin_eventsUnsubscribe")]
 	fn events_unsubscribe(&self, _: Option<Self::Metadata>, _: SubscriptionId) -> RpcResult<bool>;
+
+	#[rpc(name = "creditcoin_hashrate")]
+	fn mining_stats(&self) -> RpcResult<MiningStats>;
```

Comment:
This will need a simple integration test, see `integration-tests/tests/rpc/creditcoin.spec.ts`.